PocketKit/Sources/PocketKit/MyList/SortFilterMenu/View/SortMenuHeaderView.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+class SortMenuHeaderView: UITableViewHeaderFooterView {
+    static let identifier = "SortMenuHeaderView"
+    private lazy var sortIconImageView: UIImageView = {
```

Comment:
Is there a reason why these properties need to be `lazy`?

Reply:
Initially I wrote that way and find later if a view is needed all the time or not and adjust the code accordingly. Since we need the View elements to appear all the time, this can be removed.